src/ForkCMS/Bundle/InstallerBundle/Service/InstallerConnectionFactory.php

```diff
@@ -33,7 +34,7 @@ public function createConnection(
 
             //continue with regular connection creation using new params
             return parent::createConnection($params, $config, $eventManager, $mappingTypes);
-        } catch (ConnectionException $e) {
+        } catch (ConnectionException | DBALException $e) {
```

Comment:
TIL: very nice!

Reply:
Yup, new since PHP 7.1 
https://wiki.php.net/rfc/multiple-catch

Reply:
Sadly I'm not working on PHP7 atm 😢